src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -34,13 +35,16 @@ public static int Main(string[] args)
 
             app.OnExecute(() =>
             {
-                if (!CheckArg(framework))
+                if (framework.HasValue() && IsUnsupportedFramework(framework.Value()))
```

Comment:
`NuGetFramework` already has an [`IsUnsupported`](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs#L290) property, and we need to parse it anyway if it has a value. Can we just use that?